.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: DAML Feature request
+about: Suggest an idea to improve DAML
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A description of what the problem is --- "I'm always frustrated when [...]"
```

Comment:
```suggestion
It's really helpful when we understand _why_ you're making a feature request - for example, because it solves a problem you're having. Include A description of the problem here --- eg "I'm always frustrated when [...]"
```